setup.py

```diff
@@ -12,6 +12,10 @@
     'pyramid',
     'pyramid_jinja2',
     'pyramid_debugtoolbar',
+    'pyramid_tm',
+    'SQLAlchemy',
+    'transaction',
+    'zope.sqlalchemy',
     'waitress',
```

Comment:
If you're using the `pytz` package in your learning journal, you need to include it here.